README.md

```diff
@@ -60,5 +60,12 @@ Run the following after cloning the repo:
 ```sh
 mix deps.get
 docker-compose up -d
+DB_PORT=54323 mix test
```

Comment:
Could you revert this change about the database?

- everybody should have access to docker, it shouldn't be a blocker from contributing here
  - docker should be preferred to using a custom database, as docker is 100% controllable. if you use your own database, and you have a problem with it, we can't tell if it's caused by your setup or the library
  
- if we put 5432 as default I'm afraid it may conflict with somebody else's machine (especially since we use `postgres` user and db name as the default)
  - avoiding such conflicts should be a top priority

Reply:
>everybody should have access to docker, it shouldn't be a blocker from contributing here
It's your choice of course, but this is a bit unorthodox and might dissuade some people from contributing. Take a look at Ecto, EctoSQL and all the database drivers. Their codebases are much more complex than this one and have been accepting contributions for years without docker.